docs/specs/markdown.yml

```diff
@@ -342,7 +342,7 @@ outputs:
   docs/d.json:
   .errors.log: |
     ["warning","external-bookmark-not-found","Cannot find bookmark '#title-3' in 'docs/a.md', did you mean '#title-1'?","docs/b.md",2,1]
-    ["suggestion","internal-bookmark-not-found","Cannot find bookmark '#title-3' in 'docs/a.md', did you mean '#title-1'? NOTE: This Suggestion will be elevated to a Warning on 8/26/2019. Please fix invalid bookmarks as soon as possible.","docs/a.md",4,1]
+    ["warning","internal-bookmark-not-found","Cannot find bookmark '#title-3' in 'docs/a.md', did you mean '#title-1'?","docs/a.md",4,1]
```

Comment:
If they are both warnings now, can we consolidate the error code be the same `bookmark-not-found`?

Reply:
Updated.

Reply:
This requires the diff tool to be updated too or it may influence out monitored repo states?

Reply:
This is **behavior** change - we may need to confirm with PM first before running it. My personal suggestion would be still using two different log codes as separate log codes could be helpful to BI.

Reply:
@yufeih @mingwli @herohua Reverted the behavior change on log code, We can open a new PR for this after confirmation.